app/views/finders/_facet_collection.html.erb

```diff
@@ -7,7 +7,8 @@
     value: @results.user_supplied_keywords,
     id: "finder-keyword-search",
     controls: "js-search-results-info"
-  } if finder.show_keyword_search? %>
+  } if finder.show_keyword_search? and finder.name != 'All content' %>
```

Comment:
`finder.name != 'All content'` isn't an ideal check, as someone could change the name in the finder config, not realising it would break this logic. Ideally this would be set somewhere in the finder config, but we could also check against the content id as this is less likely to change.

Reply:
Thanks for suggestion the content id, that seems like a good alternative. As you say anyone could change the name.